### ISSUE TITLE
Unable to run release cycle for v1.3.3

### ISSUE THREAD

**Issue by cmdcolin** [GMOD/jbrowse-components#2180]
I'm wondering if it is related to  this commit  5c11c1dc6c

The hunch:

When I look at the full log there are many "test completes" and then when it hits CLI it does somethjing a little different and then there is a bunch of stuff

![Screenshot from 2021-08-02 15-24-01](https://user-images.githubusercontent.com/6511937/127912864-c75e7c13-b2e7-4ea4-a9d3-010f7e307922.png)

Attached full log
[myhist2.txt.gz](https://github.com/GMOD/jbrowse-components/files/6919185/myhist2.txt.gz)

**Comment by cmdcolin** [comment]
For the @jbrowse/cli build

```
> @jbrowse/cli@1.3.3 prepack /home/cdiesh/src/jbrowse-components2/products/jbrowse-cli
> npm-run-all clean build:typescript manifest docs
 
lerna notice cli v3.22.0
lerna info Executing command in 2 packages: "yarn run clean"
lerna info run Ran npm script 'clean' in '@jbrowse/cli' in 0.2s:
$ rimraf lib
lerna info run Ran npm script 'clean' in '@jbrowse/core' in 0.2s:
$ rimraf dist
lerna success run Ran npm script 'clean' in 2 packages in 0.2s:
lerna success - @jbrowse/core
lerna success - @jbrowse/cli
lerna notice cli v3.22.0
lerna info Executing command in 1 package: "yarn run build:typescript"
lerna info run Ran npm script 'build:typescript' in '@jbrowse/cli' in 2.4s:
$ tsc --build
lerna success run Ran npm script 'build:typescript' in 1 package in 2.4s:
lerna success - @jbrowse/cli
lerna notice cli v3.22.0
lerna info Executing command in 1 package: "yarn run manifest"
lerna info run Ran npm script 'manifest' in '@jbrowse/cli' in 0.9s:
```

It seems maybe like it is unexpected that it is cleaning out core to me there?

**Comment by cmdcolin** [comment]
note my github auth token from the paste was pasted but its revoked now so no fuss

**Comment by cmdcolin** [comment]
technically fixed by 3704a29eabca19af0e8ca91a4a61406f004e8adc

this was a weird bug, maybe a combo of lerna and npm-run-all, possibly with prepack stage in cli that we wouldn't normally observe in just normal builds

**Comment by cmdcolin** [comment]
this can be closed but if there are any follow on issues we should make lemme know
